### PR TITLE
Add two buttons to go to home or edit after submission #90

### LINKED ISSUE #90: Add two buttons to go to home or edit after submission
**Is your feature request related to a problem? Please describe.**
- Currently, we don't show any results after submitting the application. we need to show a result containing 2 buttons for go to the home and to go to the edit page

**Describe the solution you'd like**
- Add two buttons to go to home or edit after submission

Design - https://ant.design/components/result/

image

### PR DESCRIPTION
**Pull request by kumuditha-udayanga**
## Purpose

The purpose of this PR is to fix #90 

## Goals

- To add two buttons to go to home or edit application once the application is submitted.
## Approach

- Create the result component in a separate file.
- Import the result component to MentorApplication.
- Once the user submits the application if successful, it displays the result component(from Ant Design) with a home and an edit application button.

## Screenshots

Screenshot 2021-05-09 at 19 37 09

https://user-images.githubusercontent.com/27630091/117575154-34dccf00-b0fe-11eb-9428-c94868feb90d.mov

##  Checklist
- [x] This PR doesn't commit any keys, passwords, tokens, usernames, or other secrets.
- [x] I have read and understood the development best practices guidelines ( http://bit.ly/sef-best-practices )
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

## Test environment
 
macOS BigSur
IntelliJ IDEA

### CODE REVIEW

File: src/scenes/Home/components/ResultScreen/index.tsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+        extra={[
+          <Button
+            type="primary"
+            key="console"
+            onClick={() => history.push('/home')}
+          >
+            Go Home
+          </Button>,
+          <Button
+            key="buy"
```
Comment (anjula-sack) [inline src/scenes/Home/components/ResultScreen/index.tsx]: let's change the key

Comment (kumuditha-udayanga) [reply src/scenes/Home/components/ResultScreen/index.tsx]: Sure

File: src/scenes/Home/components/ResultScreen/index.tsx [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+const result = (props) => {
+  const id = props.programId;
+  const history = useHistory();
+  return (
+    <>
+      <Result
+        className={styles.result}
+        status="success"
+        title="Successfully Applied!"
+        subTitle=""
```
Comment (anjula-sack) [inline src/scenes/Home/components/ResultScreen/index.tsx]: Let's remove this

File: src/scenes/Home/components/ResultScreen/index.tsx [hunk 3]
```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { Result, Button } from 'antd';
+import styles from './styles.css';
+import { useHistory } from 'react-router-dom';
+
+const result = (props) => {
+  const id = props.programId;
```
Comment (anjula-sack) [inline src/scenes/Home/components/ResultScreen/index.tsx]: `const {programId} = props` let's use it like this

File: src/scenes/Home/components/ResultScreen/styles.css [hunk 4]
```diff
@@ -0,0 +1,4 @@
+.result {
+    margin-bottom: 160px;
+    margin-top: 130px;
+}
```
Comment (anjula-sack) [inline src/scenes/Home/components/ResultScreen/styles.css]: EOL

**Review by anjula-sack — CHANGES_REQUESTED** [review]
Good job! @kumuditha-udayanga Can we add the result inside a card?

**Comment by kumuditha-udayanga** [comment]
> Good job! @kumuditha-udayanga Can we add the result inside a card?
Thank you, sure I'll add it inside a card.

**Review by anjula-sack — APPROVED** [review]
Thank you! @kumuditha-udayanga

**Pull request merged**